src/server/router/page.js

```diff
@@ -0,0 +1,109 @@
+import config from "../config";
+import React from "react";
+import ReactDOMServer from "react-dom/server";
+import headerProps from "./componentProps/header";
+
+export default class Page {
+	constructor(app, details, rootComponent, props = {}) {
+		this.details = details;
+		this.rootComponent = rootComponent;
+
+		// Add default props
+		this.props = props;
+		this.props.serverSideProps = this.props.serverSideProps || {};
```

Comment:
Good point, on the RootComponent, this will always be the only property. When I wrote this, I still imagined we could have other props on it as well. I'll probably change this to simply use props then.